### PR TITLE
Feature/UI enhancements for Desktop Wallet

### PR DESCRIPTION
**Pull request by fmtabbara**

### CODE REVIEW

**Comment by tommyv1987** [comment]
Nice stuff @fmtabbara  - I've been through the wallet on all OS. I'm happy with these changes.

**Review by tommyv1987 — APPROVED** [review]
I'll go through and sort the automation parts out later.

File: nym-wallet/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 80,
```
Comment (mmsinclair) [inline nym-wallet/.prettierrc]: Maybe make this wider? Say `120`?

**Review by mmsinclair — APPROVED** [review]

**Comment by mmsinclair** [comment]
Be good if you can also upgrade all the packages, but if stuff breaks, then we can leave it for another time - MUI 5.2.1 fixes the overlay error

**Pull request merged**